src/Servers/Kestrel/test/FunctionalTests/ResponseTests.cs

```diff
@@ -297,7 +298,7 @@ public async Task WritingToConnectionAfterUnobservedCloseTriggersRequestAbortedT
                 {
                     for (var i = 0; i < 1000; i++)
                     {
-                        await context.Response.Body.WriteAsync(scratchBuffer, 0, scratchBuffer.Length, context.RequestAborted);
```

Comment:
So we lost a bunch of coverage for Response.Body?

Reply:
Yeah. This does lose some coverage for possible regressions in Response.Body. Fortunately, the stream implementation itself is very stable, and most of the logic being tested is in HttpProtocol or one of the OutputProduces which is used by both the Pipe and the Stream.

We could do something similar to what I did for HTTPS and run a bunch of tests twice: once with the body Stream and once with the BodyPipe, but I don't think it's worth it to do that en masse right now. If in the future, we do find ways in with the Stream and Pipe behavior diverge, we should add regression tests for both to ensure they both behave as expected.

Reply:
Yes we did. I'll quickly add some regression tests.